### DISCUSSION TITLE
wanting to save a filter made from (1) clicking multiple nodes then (2) using Filter Selected Nodes?

### DISCUSSION THREAD

**Discussion by kenfhill84083** [freeplane/freeplane#1194] [Help]
I would like to (1) click on multiple nodes and (2) filter them using Filter Selected Nodes - then **(3) save that filter to use again later**.

Is that doable?  Compose Filter on the Filter toolbar seems too structured for this.  At least I cannot figure out a way to use Compose Filter for what I am describing.

Thanks.

**Comment by macmarrum** [comment]
I can only think of a script that could do it. Freeplane core lacks such functionality.
